tests/README.md

```diff
@@ -20,9 +20,9 @@ npx ava scalers/prometheus.test.ts
 ## E2E test setup
 
 The test script will run 3 phases:
-- **Setup**: this is done in [`setup.test.ts`](setup.test.ts). If you're adding any tests for KEDA install/setup process add it to this file.`setup.test.ts` deploys [`/deploy/KedaScaleController.yaml`](../deploy/KedaScaleController.yaml) to `keda` namespace in the cluster, and updates the image to `kedacore/keda:master`
+- **Setup**: this is done in [`setup.test.ts`](setup.test.ts). If you're adding any tests for KEDA install/setup process add it to this file.`setup.test.ts` deploys keda to `keda` namespace in the cluster, and updates the image to `kedacore/keda:master`.
```

Comment:
```suggestion
- **Setup**: this is done in [`setup.test.ts`](setup.test.ts). If you're adding any tests for KEDA install/setup process add it to this file.`setup.test.ts` deploys KEDA to `keda` namespace in the cluster, and updates the image to `kedacore/keda:main`.
```

Reply:
oh right. wonder how i skipped it. will edit it @zroubalik . thanks.

Reply:
Done. @zroubalik  PTAL